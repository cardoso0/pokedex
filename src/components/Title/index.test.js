import { render, screen } from "@testing-library/react"
import "@testing-library/jest-dom"
import renderer from "react-test-renderer"

import { Title } from "."

const renderTitle = 
<Title />

describe("Title Component", () => {
  it ("component rendering | snapshot", () => {
    const tree = renderer
    .create(renderTitle)
    .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it ("component rendering", () => {
    expect(render(renderTitle)).toBeTruthy
  })
})