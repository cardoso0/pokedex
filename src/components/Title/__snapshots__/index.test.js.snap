// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component component rendering | snapshot 1`] = `
<section
  className="sc-bczRLJ bHndnD"
>
  <h1 />
  <h2 />
</section>
`;
